src/cmd/troubleshoot/config.go

```diff
@@ -0,0 +1,19 @@
+package troubleshoot
+
+import (
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+type TestData struct {
```

Comment:
does it have to be exported ?

you use it as `troubleshootContext` which is a far better name

the name of the struct could be `troubleshootContext` and the variables could be called `troubleshootCtx` so there is no naming clash

Reply:
couldn't the `apiReader client.Reader` be part of this struct as well ?